src/Routers/FeaturesRouter.js

```diff
@@ -20,10 +20,10 @@ export class FeaturesRouter extends PromiseRouter {
     if (!req.auth.isMaster) {
       return masterKeyRequiredResponse();
     }
-    
+
     return Promise.resolve({
       response: {
-        results: [getFeatures()]
+        results: getFeatures()
```

Comment:
Also suppose there were a feature named `error` and another named `code`, it would be impossible to determine success vs. failure